share/metainfo/io.github.ladaapp.lada.metadata.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>io.github.ladaapp.lada</id>
+
+    <name>lada</name>
+    <summary>Remove and recover pixelated content in adult videos</summary>
+
+    <metadata_license>MIT</metadata_license>
+    <project_license>AGPL-3.0-or-later</project_license>
+
+    <description>
+        <p>Remove and recover pixelated content in adult videos</p>
+        <p>Features</p>
+        <ul>
+            <li>Remove and recover pixelated content in adult videos</li>
+            <li>Watch or convert your videos via CLI or GUI</li>
+        </ul>
+    </description>
+
+    <url type="homepage">https://github.com/ladaapp/lada</url>
+
+    <launchable type="desktop-id">io.github.ladaapp.lada.desktop</launchable>
+
+    <screenshots>
+        <screenshot type="default">
+            <image>https://github.com/ladaapp/lada/blob/main/assets/screenshot_gui_1.png</image>
+            <caption>Watching restored video</caption>
+        </screenshot>
+        <screenshot type="default">
+            <image>https://github.com/ladaapp/lada/blob/main/assets/screenshot_gui_2.png</image>
+            <caption>Exporting restored video</caption>
+        </screenshot>
+        <screenshot type="default">
+            <image>https://github.com/ladaapp/lada/blob/main/assets/screenshot_cli_1.png</image>
+            <caption>Exporting restored video via CLI</caption>
+        </screenshot>
+    </screenshots>
+</component>
```

Comment:
missing mandatory `releases` tag

Reply:
sure will add a release in upstream then